external/qjs_fs_module.c

```diff
@@ -2689,13 +2689,6 @@ qjs_fs_filehandle_finalizer(JSRuntime *rt, JSValue val)
 }
 
 
-static JSValue
-qjs_fs_promise_trampoline(JSContext *cx, int argc, JSValueConst *argv)
-{
-    return JS_Call(cx, argv[0], JS_UNDEFINED, 1, &argv[1]);
-}
-
-
 static JSValue
 qjs_fs_result(JSContext *cx, JSValue result, int calltype, JSValue callback)
```

Comment:
The `result` might be `JS_EXCEPTOIN`, then is it an issue about this.
```
arguments[1] = result;
```
It's not ERROR CLASS.
I don't know how to test it, I think we can test it with JS throw and check if `e.message` works.